app/javascript/packs/cms.ts

```diff
@@ -332,14 +312,25 @@ function setUpDropdownButtonClose () {
 }
 
 function closeAllDropdowns (exception?: HTMLElement) {
-  document.querySelectorAll('.dropdown.expanded')
+  document.querySelectorAll('.pf-c-dropdown.pf-m-expanded')
     .forEach(dropdown => {
-      const buttonGroup = dropdown.parentElement as HTMLDivElement
-
-      if (exception && buttonGroup.contains(exception)) {
+      if (exception && dropdown.contains(exception)) {
         return
       }
-
-      dropdown.classList.remove('expanded')
+      dropdown.classList.remove('pf-m-expanded')
+      dropdown.querySelector("ul")!.style.display = 'none'
     })
 }
+
+/**
+ * Set up CMS Preview button to act as a link
+ */
+function setUpPreviewButton () {
+  $(document).on('click', '#cms-preview-button button.pf-c-dropdown__toggle-button:not(.dropdown-toggle)', (event) => {
+    const url = event.target.dataset.url
+    if (url) {
+      window.open(url)
```

Comment:
Open in a new tab, maybe?
```suggestion
      window.open(url, 'previewTab')
```

Reply:
hmm, what's `previewTab` ? :thinking: I can see `_self`, `_blank` and others here: https://developer.mozilla.org/en-US/docs/Web/API/Window/open#target but not `previewTab`. 

The default is `_blank` which already opens in a new tab.

Reply:
I thought you needed to give a name to the new tab, I was wrong.